quickstarts/dotnet/csharp/config.yml

```diff
@@ -2,20 +2,26 @@ id: 9b575bdc-5bf0-450f-a041-8462b886d5ee
 
 slug: c-sharp
 description: |
-  ## Comprehensive monitoring quickstart for C#
+  ## C# performance monitoring
   With our C# integration, you can find and address performance bottlenecks while operating in a live development environment. Monitor your app via our dashboard quickstarts and ensure you're delivering the best user experience.
-  
-  ## Why monitor C#?
+    
+  ### Common C# performance errors
+  Here are the common C# performance errors that you can handle with New Relic integration:
+  - Memory management: C# uses automatic memory management, which takes the burden of manual allocation from developers. However, C# can sometimes run into memory error thereby inhibiting its memory management performance. Our integration offers the best solution to fix that.
+  - CPU usage: New Relic’s high CPU utilization alert is the key to monitoring CPU usage to prevent any potential error.
+  - Garbage collection: Garbage Collector (GC) manages the allocation and release of memory for your application. When the memory is under distress, our C# integration will easily notify you via the memory usage alert.
+  - High volume of requests: Getting a high volume of requests can affect your app’s response time and reduce user satisfaction. With Apdex score, you can track requests and measure user satisfaction.
+  - JIT compiler: If JIT compiler fails to load, it may be due to an out-of-memory exception or internal limitation error. The pathway towards getting the insights you need is the New Relic C# integration.
+
+  ### C# monitoring use cases
   - Focus on the issues that affect your critical business transactions.
-  - Resolve problems quickly.
-  - Gain knowledge on your system's behavior in advance.
-  - Learn the whole story of performance errors, from beginning to end.
+  - Gain knowledge on your system's behavior in advance and learn the whole story of performance errors, from beginning to end.
   - Use traces to connect latency issues to errors, getting you to the root cause as quickly as possible.
   - See the filename and line number from the C# stack trace so you never have to guess. To reduce noise, logically filter and aggregate C# exceptions.
   - Scalable error monitoring without affecting production flow.
   - Maintain low latency with fast throughput.
-  - Get meaningful data about C# errors from your dashboard.
-  
+ -  Get meaningful data about C# errors from your dashboard.
```

Comment:
Looks like we removed some indentation here - this line should be at the same indentation as the rest of the description

Reply:
```suggestion
  -  Get meaningful data about C# errors from your dashboard.
```